tests/system_tests_sasl_plain.py

```diff
@@ -22,15 +22,31 @@
 from __future__ import absolute_import
 from __future__ import print_function
 
-import os
-from subprocess import PIPE, Popen
-from system_test import TestCase, Qdrouterd, main_module, DIR, TIMEOUT, SkipIfNeeded
+import os, json
+from subprocess import PIPE, STDOUT, Popen
+from system_test import TestCase, Qdrouterd, main_module, DIR, TIMEOUT, SkipIfNeeded, Process
 from system_test import unittest
 from qpid_dispatch.management.client import Node
 from proton import SASL
 
 class RouterTestPlainSaslCommon(TestCase):
 
+    def run_qdmanage(self, cmd, input=None, expect=Process.EXIT_OK,
```

Comment:
We've cut and pasted this code too many times.
See system_test.py - class QdManager().   It wraps qdmanage and exports operations as methods.  See system_tests_edge_router.py for an example.